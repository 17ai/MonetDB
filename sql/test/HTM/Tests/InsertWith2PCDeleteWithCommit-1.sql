START TRANSACTION;
INSERT INTO htmtest VALUES (16, 99), (17, 99), (18, 99);
SELECT * FROM htmtest;
CALL precommit(6);
CALL persistcommit(6);
SELECT * FROM htmtest;

DELETE FROM htmtest WHERE id > 3;
SELECT * FROM htmtest;
