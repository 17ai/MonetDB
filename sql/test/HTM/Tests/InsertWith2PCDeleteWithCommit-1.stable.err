stderr of test 'InsertWith2PCDeleteWithCommit-1` in directory 'sql/test/HTM` itself:


# 17:16:33 >  
# 17:16:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38035" "--set" "mapi_usock=/var/tmp/mtest-1912/.s.monetdb.38035" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/Users/dnedev/monetdb/installation/var/MonetDB/mTests_sql_test_HTM" "--set" "mal_listing=0"
# 17:16:33 >  

# builtin opt 	gdk_dbpath = /Users/dnedev/monetdb/installation/var/monetdb5/dbfarm/demo
# builtin opt 	gdk_debug = 0
# builtin opt 	gdk_vmtrim = no
# builtin opt 	monet_prompt = >
# builtin opt 	monet_daemon = no
# builtin opt 	mapi_port = 50000
# builtin opt 	mapi_open = false
# builtin opt 	mapi_autosense = false
# builtin opt 	sql_optimizer = default_pipe
# builtin opt 	sql_debug = 0
# cmdline opt 	gdk_nr_threads = 0
# cmdline opt 	mapi_open = true
# cmdline opt 	mapi_port = 38035
# cmdline opt 	mapi_usock = /var/tmp/mtest-1912/.s.monetdb.38035
# cmdline opt 	monet_prompt = 
# cmdline opt 	mal_listing = 2
# cmdline opt 	gdk_dbpath = /Users/dnedev/monetdb/installation/var/MonetDB/mTests_sql_test_HTM
# cmdline opt 	mal_listing = 0
# cmdline opt 	gdk_debug = 536870922

# 17:16:33 >  
# 17:16:33 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1912" "--port=38035"
# 17:16:33 >  


# 17:16:33 >  
# 17:16:33 >  "Done."
# 17:16:33 >  

