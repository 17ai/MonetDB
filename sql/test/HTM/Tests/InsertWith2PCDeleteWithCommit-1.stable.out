stdout of test 'InsertWith2PCDeleteWithCommit-1` in directory 'sql/test/HTM` itself:


# 17:16:33 >  
# 17:16:33 >  "mserver5" "--debug=10" "--set" "gdk_nr_threads=0" "--set" "mapi_open=true" "--set" "mapi_port=38035" "--set" "mapi_usock=/var/tmp/mtest-1912/.s.monetdb.38035" "--set" "monet_prompt=" "--forcemito" "--set" "mal_listing=2" "--dbpath=/Users/dnedev/monetdb/installation/var/MonetDB/mTests_sql_test_HTM" "--set" "mal_listing=0"
# 17:16:33 >  

# MonetDB 5 server v11.22.0
# This is an unreleased version
# Serving database 'mTests_sql_test_HTM', using 4 threads
# Compiled for x86_64-apple-darwin14.3.0/64bit with 64bit OIDs and 128bit integers dynamically linked
# Found 8.000 GiB available main-memory.
# Copyright (c) 1993-July 2008 CWI.
# Copyright (c) August 2008-2015 MonetDB B.V., all rights reserved
# Visit http://www.monetdb.org/ for further information
# Listening for connection requests on mapi:monetdb://MacBookAir.local:38035/
# Listening for UNIX domain connection requests on mapi:monetdb:///var/tmp/mtest-1912/.s.monetdb.38035
# Start processing logs sql/sql_logs version 52200
# Start reading the write-ahead log 'sql_logs/sql/log.10'
# Finished reading the write-ahead log 'sql_logs/sql/log.10'
# Finished processing logs sql/sql_logs
# MonetDB/SQL module loaded

Ready.

# 17:16:33 >  
# 17:16:33 >  "mclient" "-lsql" "-ftest" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-1912" "--port=38035"
# 17:16:33 >  

#START TRANSACTION;
#INSERT INTO htmtest VALUES (16, 99), (17, 99), (18, 99);
[ 3	]
#SELECT * FROM htmtest;
% sys.htmtest,	sys.htmtest # table_name
% id,	val # name
% int,	int # type
% 2,	2 # length
[ 1,	10	]
[ 2,	20	]
[ 3,	88	]
[ 16,	99	]
[ 17,	99	]
[ 18,	99	]
#CALL precommit(6);
[ 1	]
#CALL persistcommit(6);
[ 1	]
#SELECT * FROM htmtest;
% sys.htmtest,	sys.htmtest # table_name
% id,	val # name
% int,	int # type
% 2,	2 # length
[ 1,	10	]
[ 2,	20	]
[ 3,	88	]
[ 16,	99	]
[ 17,	99	]
[ 18,	99	]
#DELETE FROM htmtest WHERE id > 3;
[ 3	]
#SELECT * FROM htmtest;
% sys.htmtest,	sys.htmtest # table_name
% id,	val # name
% int,	int # type
% 1,	2 # length
[ 1,	10	]
[ 2,	20	]
[ 3,	88	]

# 17:16:33 >  
# 17:16:33 >  "Done."
# 17:16:33 >  

